packages/config/test/methods.test.ts

```diff
@@ -51,6 +51,16 @@ describe("when it can't find a config file", () => {
       TruffleConfig.detect();
     }, "should have thrown!");
   });
+
+  it("doesn't throw when url is provided", () => {
+    const options = { url: "http://localhost:7545" };
+    const config = TruffleConfig.detect(options);
+    const networks = config.networks
```

Comment:
This is me being really nitpicky, but, missing semicolon here.  Btw, we now have `prettier` and `eslint` on this repo (dunno when you forked it), so if you run `yarn bootstrap` after cloning it to set things up, `prettier` will automatically fix things like this for you. :)